spec/lib/main_class_generator_spec.rb

```diff
@@ -31,17 +31,85 @@
       exercise.testIO = io
       exercise.code = 'return "asdasdasd";'
 
+      method = <<-eos
+    public static String metodi(String input) {
```

Comment:
Yes, it's (at least at the moment) too difficult to parse any random argument name a student may come up with from the code.